agent/functional_tests/tests/generated/simpletests_unix/simpletests_generated_unix_test.go

```diff
@@ -397,7 +397,7 @@ func TestInitProcessEnabled(t *testing.T) {
 	}
 	agent := RunAgent(t, nil)
 	defer agent.Cleanup()
-	agent.RequireVersion(">=1.14.5")
+	agent.RequireVersion(">=1.15.0")
```

Comment:
Why did this change?

Reply:
I believe this was changed to `1.14.5` unexpectedly [here](https://github.com/aws/amazon-ecs-agent/commit/c268ea09b58d6eacf704ccf648e5ae6817a2b8a0), so I changed it back.

Reply:
@sharanyad Can you clarify whether that was intended?

Reply:
@richardpen is right. According to the releases, `1.15.0` has the `init` support in agent. Not sure how the testing change got slipped in.